src/auto-trace.js

```diff
@@ -104,29 +90,19 @@ export function syncStacktrace(rawError) {
  * @param {Object} rawError
  * @throws {Error}
  */
-export function throwSyncStacktrace(rawError) {
+export function catchSyncStacktrace(rawError) {
 	const syncStacktraceErr = new Error();
 	const syncMiddlewareErrFunctions = executeAsyncMiddleware(syncStacktraceErr);
 	const middlewareErr = executeSyncMiddleware(syncMiddlewareErrFunctions, rawError)
 	const syncErr = wrapObjectWithError(middlewareErr)
-	throw syncErr;
+	setTimeout(() => {throw syncErr});
```

Comment:
This is a breaking change, but I think it's okay. It means that any code that does this:

```js
throwSyncStacktrace("error");
```
now needs to be
```js
throwSyncStacktrace("error");
return; // Assuming that returning undefined is okay
```